test/integration/util/util.go

```diff
@@ -81,6 +81,18 @@ func (m *MinishiftRunner) Start() {
 	m.RunCommand(fmt.Sprintf("start %s", m.CommandArgs))
 }
 
+func (m *MinishiftRunner) CDKSetup() {
+	m.RunCommand(fmt.Sprintf("setup-cdk --force --minishift-home %s", os.Getenv(constants.MiniShiftHomeEnv)))
+}
+
+func (m *MinishiftRunner) IsCDK() bool {
+	cmdOut, _, _ := m.RunCommand("-h")
+	if strings.Contains(cmdOut, "setup-cdk") {
```

Comment:
Also rather see:

```
+                      if (os.Getenv("MINISHIFT_USERNAME") == "") || (os.Getenv("MINISHIFT_PASSWORD") == "") {
 +				fmt.Println("Either MINISHIFT_USERNAME or MINISHIFT_PASSWORD is not set as environment variable")
 +				os.Exit(1)
 +			}
```
here instead of the before suite. as it reduces the nesting in `BeforeSuite`, and is easier to reason about if the error occurs.